runtime/error.c

```diff
@@ -18,6 +18,15 @@
 #define EXCEEDS_MAX_PC_COUNT -3
 #define OUT_OF_MEMORY -4
 
+#define PUBLIC_BALLERINA_NAME 1
```

Comment:
These numbers seem fairly random to me. Simpler just to make DemangleResult be an enum.

Reply:
Fixed